api/python/cellxgene_census/src/cellxgene_census/_release_directory.py

```diff
@@ -158,38 +200,64 @@ def get_census_version_directory() -> Dict[CensusVersionName, CensusVersionDescr
         'h5ads': {'uri': 's3://cellxgene-data-public/cell-census/2022-11-29/h5ads/',
         's3_region': 'us-west-2'}}}
     """
+    if invalid_release_flags := release_flags_kwargs.keys() - RELEASE_FLAGS:
+        raise ValueError(f"Unknown release flag(s): {invalid_release_flags}")
+
     response = requests.get(CELL_CENSUS_RELEASE_DIRECTORY_URL)
     response.raise_for_status()
 
     directory: CensusDirectory = cast(CensusDirectory, response.json())
+    directory_out: CensusDirectory = {}
+    aliases: Dict[CensusVersionName, List[CensusVersionName]] = {}
```

Comment:
much of the below code changes are for handling alias mappings internally in the method, to avoid adding a synthentic "alias" key/value to each CensusVersionDescription.

Reply:
Can you comment on the motivation for this change?  I'm not opposed, but also don't see any functional benefit (I'm probably missing something obvious!)

Reply:
This is _only_ needed later in the method to order the output, where it needs to know which releases are identified by aliases. Realizing now that this can be simplified, so I've now done that.